README.md

```diff
@@ -140,3 +140,5 @@ Please collect logs by following these steps to append to the report:
 ## Release Notes
 
 See the [CHANGELOG](https://github.com/google/vscode-zxdb/blob/main/CHANGELOG.md)
+
+Note: This is not an officially supported Google product
```

Comment:
It might be better to put it at the beginning of the README, e.g. first or second paragraph.

Reply:
Ack